BSP/Simulator/Src/BSP_Fans.c

```diff
@@ -0,0 +1,62 @@
+#include "BSP_Fans.h"
+#include "simulator_conf.h"
+#include <unistd.h>
+#include <sys/file.h>
+
+static const char* file = GET_CSV_PATH(FANS_CSV_FILE);
+static char Fans[5];
+/**
+ * @brief   Initialize all the GPIO pins connected to each fan
+ * @param   None
+ * @return  None
+ */
+void BSP_Fans_Init(void)
+{
+    // Creates file if none exists
+    FILE* fp = fopen(file, "w+");
+    int fno = fileno(fp);
+    flock(fno, LOCK_EX);
+    fprintf(fp, "0000"); 

```

Comment:
Is each digit supposed to represent a fan? Since this is a csv file, I think it would make more sense if the duty cycles were separated by commas. This would also allow us to have more than ten duty cycle settings in the future.